ai-platform/snippets/test/predict-text-embeddings.test.js

```diff
@@ -48,7 +48,7 @@ describe('predict text embeddings', () => {
   });
   it('should get text embeddings using the preview model', async () => {
     const stdout = execSync(
-      `node ./predict-text-embeddings-preview.js ${project} text-embedding-preview-0409 '${texts.join(';')}' QUESTION_ANSWERING ${dimensionality}`,
```

Comment:
Don't pass command line arguments. See the other comment I left about just setting variables in the sample to demonstrate how it's supposed to work.